### PR TITLE
Rails Blog Project: Add form

### PR DESCRIPTION
**Pull request by Profsain**
### I completed this part of Rails Blog project by implementing the following requirement:
- Created current user method to fetch the first user
- Created form
- Created comment and likes post function

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  def create
```
Comment (tufoinnkuo10) [inline app/controllers/comments_controller.rb:4]: - [ ] great job adding the creation action. i am not able to create a new comment. please kindly ensure the comments controller should have new and create actions. i cant find the `new` action
eg 
```ruby
def new
    @comment = Comment.new
  end
```

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts
   end
 
   def show
+    @post = @user ? @user.posts.find(params[:id]) : Post.find(params[:id])
+  end
+
+  def new; end
```
Comment (tufoinnkuo10) [inline app/controllers/posts_controller.rb:14]: - [ ] kindly add a new post action..
eg
```ruby
def new
      @post = Post.new
    end
```

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Profsain 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (Profsain) [reply app/controllers/posts_controller.rb:14]: Thank you for that feedback

**Review by julie-ify — APPROVED** [review]
Hi @Profsain,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**